### DISCUSSION TITLE
about workflow context

### DISCUSSION THREAD

**Discussion by Nokecy** [elsa-workflows/elsa-core#1523] [Q&A]
I want to update the status of the business entity at the end of the process, is it possible to use the workflow context

**Comment by sfmskywalker** [comment]
Yes, one easy way to update your business entity is to use the RunJavaScript activity wit code like this:

```javascript
workflowContext.SomeProperty = "some value";
```

If you have your workflow context provider class in place (by inheriting from `WorkflowContextRefresher<MyBusinessEntity>`) then your code will be invoked to handle the persistence of the business entity.

**Reply by Nokecy** [reply]
`WorkflowContextRefresher<MyBusinessEntity>`  when to call  ~~~~~~ @sfmskywalker

**Reply by sfmskywalker** [reply]
You only need to create and implement that class and register it with DI. The workflow engine will then invoke it for you.
